vignettes/function-flow.Rmd

```diff
@@ -97,98 +97,234 @@ library(dplyr)
 library(osdc)
 ```
 
-#### HbA1c tests above the diagnosis cut-off value (48 mmol/mol or 6.5%)
+#### Hospital diagnoses
 
-The function `include_hba1c()` uses `lab_forsker` as the input data to
-extract all events of HbA1c tests above the diagnosis cut-off value.
+**Joining LPR2 and LPR3 data**
 
-Since the HbA1c diagnosis cut-off value depends on the kind of test that is
-used, the inclusion event is defined as follows:
+The helper functions `join_lpr2()` and `join_lpr3()` join records of
+diagnoses to administrative information in LPR2-formatted and
+LPR3-formatted data, respectively.
 
--   For HbA1c IFCC (NPU03835), we include values \>= 6.5 %.
--   For HbA1c DCCT (NPU27300), we include values \>= 48 mmol/mol.
+`join_lpr2()` takes `lpr_diag` and `lpr_adm` as inputs, filters to the
+necessary diagnoses (`c_diag` starting with "DO", "DZ3", "DE1[0-4]",
+"249", or "250"), joins the required information by record number
```

Comment:
Wait, these patterns are different from the ones indicated below, like here are the extra "D0" and "DZ3". What are they used for?

Reply:
My mistake! I just put this as a placeholder while looking up the exact diagnoses, but then I must've gotten side-tracked. Anyway, the last commit fixes this, and other comments.